### PR TITLE
VIITE-1360 and VIITE-1397

### PR DESCRIPTION
**Pull request by skosonen**
VIITE-1360 Remove municipality restrictions from code
VIITE-1397 Optimize road address queries

### CODE REVIEW

File: digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala [hunk 1]
```diff
@@ -120,22 +119,22 @@ class RoadAddressService(roadLinkService: RoadLinkService, eventbus: DigiroadEve
...
                         everything: Boolean = false, publicRoads: Boolean = false): BoundingBoxResult = {
     BoundingBoxResult(
-      roadLinkService.getChangeInfoFromVVHF(boundingRectangle, municipalities),
+      roadLinkService.getChangeInfoFromVVHF(boundingRectangle, Set()),
       Future(fetchRoadAddressesByBoundingBox(boundingRectangle)),
-      Future(roadLinkService.getRoadLinksFromVVH(boundingRectangle, roadNumberLimits, municipalities, everything, publicRoads, frozenTimeVVHAPIServiceEnabled)),
-      Future(roadLinkService.getComplementaryRoadLinksFromVVH(boundingRectangle, municipalities)),
-      roadLinkService.getSuravageLinksFromVVHF(boundingRectangle, municipalities)
+      Future(roadLinkService.getRoadLinksFromVVH(boundingRectangle, roadNumberLimits, Set(), everything, publicRoads, frozenTimeVVHAPIServiceEnabled)),
+      Future(roadLinkService.getComplementaryRoadLinksFromVVH(boundingRectangle, Set())),
```
Comment (Campanha) [inline digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala]: this method already have Set() by default

Comment (Campanha) [inline digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala]: this method already have Set() by default

**Comment by skosonen** [comment]
Just noticed that I accidentally committed also VIITE-1397 changes in this branch. I guess they can be reviewed here together then...

**Comment by skosonen** [comment]
This cannot be merged yet. We have to get materialized view creation privileges for VIITE-users first.

**Comment by skosonen** [comment]
Unfortunately we cannot create materialized view from road_address-table, because it contains sdo_geometry-type column:

"You cannot create refresh-on-commit materialized views based on a master with user-defined types or Oracle-supplied types. Refresh-on-commit materialized views are those created using the ON COMMIT REFRESH clause in the CREATE MATERIALIZED VIEW statement."
https://docs.oracle.com/cd/B28359_01/server.111/b28326/repmview.htm#i55740

**Pull request closed without merging**